app/assets/stylesheets/system/admin/users.scss

```diff
@@ -0,0 +1,11 @@
+.system-admin-users {
+  &.index {
+    .user form {
+      display: flex;
+
+      .user_role {
```

Comment:
@kxmbrian actually `user_role` is generated by simple_form right? Maybe better if we define our own css classes in case their internal APIs change (although I don't expect that to happen).